CoEpi/ui/symptomReport/FollowUp/Fever/FeverViewController1.xib

```diff
@@ -0,0 +1,116 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I noticed very frequent usage of `StackView`. Not necessarily bad (and doesn't have to be changed in this PR), but consider in general avoiding adding unnecessary views to the hierarchy. Mostly it can be solved using only constraints.

Reply:
I agree. One place where they are necessary is the on boarding. I am using ".isHidden" for the join and FAQ buttons. constraints still apply to hidden buttons, but stack view spacing does not.